packages/blitz/src/cli/utils/codegen-tasks.ts

```diff
@@ -18,7 +19,7 @@ export const codegenTasks = async () => {
     const nextClientIndex = join(nextDir, "../..", "client", "index.js")
     const readFile = await fs.readFile(nextClientIndex)
     const packageJson = await getPackageJson()
-    const nextVersion = packageJson.dependencies.next
+    const nextVersion = toValidSemver(packageJson.dependencies.next)
```

Comment:
Hey @vitaliemiron, thanks for the PR! 
I like the idea of the changes to convert the nextjs version to valid semver to avoid problems like in the mentioned issue. 

However I think it would be better to use the official `semver` package to make these changes.

```suggestion
    const version = packageJson.dependencies.next
    const nextVersion = semver.clean(version, { loose: true }) || semver.valid(semver.coerce(version))
```